tsconfig.json

```diff
@@ -6,8 +6,7 @@
     "target": "ES2017",
     "outDir": "./dist",
     "allowJs": true,
-    "jsx": "react",
```

Comment:
Why did you remove `"jsx": "react"` This is not mentioned in the PR.

Reply:
Oh sorry, I should've left a comment or a separate commit but I noticed it was a duplicate import.